sherlock/sherlock.py

```diff
@@ -19,18 +19,26 @@
 
 import requests
 
+# Removing __version__ here will trigger update message for users
+# Do not remove until ready to trigger that message
+__version__ = "0.14.4"
+del __version__
```

Comment:
Since this variable is only being used for checking for updates, cant we just remove this and then check the `__init__,py` file instead?

Reply:
We could (and I forgot to make that change, so good call), but we still have to leave that text in place until ready to bump the version

If we remove it from sherlock.py now, before the next version num is ready, it'll trigger unnecessary update available notifications

Will commit a change to pull version num from __init__ in a bit

Reply:
@sdushantha Changed url to check __init__
Note that testing this feature locally will fail since it doesn't actually exist til merged